docs/serverless/06-01-function-cr.md

```diff
@@ -69,6 +69,8 @@ spec:
         type: ConfigurationReady
 ```
 
+> **CATION:** When you create a Function, the exported object in the Function's body must have `main` as a handler name.
```

Comment:
Shouldn't **handler** be put in bold here (just as in the other document)? 🤔

Reply:
As discussed, I will leave it as it is.